### PR TITLE
Functional tests - Add new tests for stores page

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add new tests for stores page
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `TEST_PATH="functional/BO/13_shopParameters/05_contact/stores/*" URL_FO=shopUrl/ npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20569)

### CODE REVIEW

File: tests/UI/pages/BO/shopParameters/stores/index.js [hunk 1]
```diff
@@ -181,6 +200,120 @@ class Stores extends BOBasePage {
...
+   * @param page
+   * @param row
+   * @param wantedStatus
+   * @return {Promise<void>}
+   */
+  async setStoreStatus(page, row, wantedStatus) {
+    const actualStatus = await this.getStoreStatus(page, row);
+
+    if (actualStatus !== wantedStatus) {
+      await this.clickAndWaitForNavigation(page, `${this.tableColumnStatus(row)} a`);
```
Comment (SimonGrn) [inline tests/UI/pages/BO/shopParameters/stores/index.js]: Shouldn't that be its own selector ?

File: tests/UI/pages/BO/shopParameters/stores/index.js [hunk 2]
```diff
@@ -181,6 +200,120 @@ class Stores extends BOBasePage {
...
+      page.click(this.selectAllLink),
+      page.waitForSelector(this.selectAllLink, {state: 'hidden'}),
+    ]);
+  }
+
+  /**
+   * Enable / disable stores by bulk actions
+   * @param page
+   * @param statusWanted
+   * @return {Promise<string>}
```
Comment (nesrineabdmouleh) [inline tests/UI/pages/BO/shopParameters/stores/index.js]:
```suggestion
   * @return {Promise<void>}
```

File: tests/UI/pages/BO/shopParameters/stores/index.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/pages/BO/shopParameters/stores/index.js]: `tableColumnStatus` was not used without `a` so i updated it.
Thanks

File: tests/UI/pages/BO/shopParameters/stores/index.js (hunk 2 above)
Comment (boubkerbribri) [reply tests/UI/pages/BO/shopParameters/stores/index.js]: Done thanks

**Review by SimonGrn — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Pull request merged**